app/main.py

```diff
@@ -1,8 +1,27 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: str, age: str) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    instance_list = []
+    for human in people:
+        instance = Person(human["name"], human["age"])
+        instance_list.append(instance)
+
+    for instance in instance_list:
+        for human in people:
```

Comment:
you dont need nested loop here (I had the same mistake and mentor requested to change it). your list of  dicts has format "name: instance" , and the same name you can pick by human["name"]. so we have: for human in people: instance_list[ human["name"]]... I saved  human["name"]  in additinal var  just for not to mix it up...